### PR TITLE
Set Heroku Deployment Configuration

### PR DESCRIPTION
**Pull request by jestradadeveloper**
## What does this change?
We set all the things required for a heroku deployment and then we add a set of html elements to view on the production url. Also we add a tailwindcss library for making styles. Header and footer created on partials files.

## Why are you changing that?
We made this change to have a graphical view for verifying something is displaying on production., in order to validate that Heroku was configured correctly. 

## Merge/Deploy Checklist
* [ ]   It has been reviewed and accepted.

## How do you manually test this?
You can go to Heroku production url: 
[Check-in Heroku Production App](https://still-dusk-94334.herokuapp.com/)

## Screenshots (if apply)
Captura de Pantalla 2021-10-01 a la(s) 11 15 32

### CODE REVIEW

File: app/views/layouts/partials/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            <li>Jose Maria</li>
+            <li>Adrian</li>
+        </ul>
+    </div>
+    <div class="container mx-auto my-4 py-4 text-center">
+        <div class="border-t border-white w-6/12 mx-auto pt-5">
+            <p class="text-white"><a href="http://www.brightcoders.com/">BrightCoders</a> | 2021 </p>
+        </div>
+    </div>
+</footer>
```
Comment (juliocabrera820) [inline app/views/layouts/partials/_footer.html.erb:16]: keep your indentation with 2 spaces

File: app/views/layouts/partials/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<header>
+    <div class="sm:container mx-auto mt-5">
+        <div class="w-8/12 mx-auto text-center border-b border-blue-500 pb-3">
+            <h1 class="text-2xl text-blue-500">Check-in App</h1>
+        </div>
+    </div>
+</header>
```
Comment (juliocabrera820) [inline app/views/layouts/partials/_header.html.erb:7]: same here

File: app/views/static/index.html.erb [hunk 3]
```diff
@@ -1,2 +1,6 @@
 
-<%= react_component("App", { greeting: "Hello" }) %>
+<div class="sm:container mx-auto mt-5">
+    <div class="w-6/12 mx-auto text-center pb-3 shadow my-5 p-5">
+        <span class="font-bold">React component example: </span> <%= react_component("App", { greeting: "Hello" }) %>
+    </div>
+</div>
```
Comment (juliocabrera820) [inline app/views/static/index.html.erb:6]: here

**Review by juliocabrera820 — COMMENTED** [review]
Just keep your indentation

File: app/views/layouts/partials/_footer.html.erb (hunk 1 above)
Comment (jestradadeveloper) [reply app/views/layouts/partials/_footer.html.erb]: Thanks for your review, i already change the indentation with 4 spaces to 2 spaces.

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by CUBOFIG — APPROVED** [review]
LGTM

**Pull request merged**